doc/site/modules/core/list.markdown

```diff
@@ -173,3 +185,13 @@ var other = ["d", "e", "f"]
 var combined = letters + other
 System.print(combined)  //> [a, b, c, d, e, f]
 </pre>
+
+### **\***(count) operator
```

Comment:
Should be two asterisks, not three.

Reply:
I thought that if you wanted a literal asterisk you had to escape it with a back-slash in Github markdown?

Reply:
Oh my bad, I thought that this is operator `\` (now I recall that the operator is reverse, `/`) 😄.

Reply:
No problem. I had 5 asterisks to begin with before I realized the middle one needed escaping!